mkt/tags/models.py

```diff
@@ -37,25 +37,23 @@ def can_reverse(self):
     def get_url_path(self):
         return reverse('tags.detail', args=[self.tag_text])
 
-    def save_tag(self, addon):
+    def _get_m2m_name(self, obj):
+        model = obj._meta.model_name
+        if model == 'webapp':
+            return 'tags'
+        elif model == 'website':
+            return 'keywords'
```

Comment:
I don't think it's that important. Would be more DRY, but it's not *that* bad. If you don't do it, maybe just add a `else` throwing an exception ? That will help us when we introduce tags to other models in the future.